docs/ref/checks.txt

```diff
@@ -664,4 +664,4 @@ The following checks are performed on your URL configuration:
   including a ``:``. Remove the colon, to avoid ambiguous namespace
   references.
 * **urls.E004**: Your URL pattern ``<pattern>`` is invalid. Ensure that
-  ``urlpatterns`` is a list of :func:`~django.conf.urls.url()` instances.
+  ``urlpatterns`` is a list of :func:`~django.urls.path()` instances.
```

Comment:
url() isn't going away, is it? I think we need to keep its documentation and make sure the docs describe it as an equally okay alternative.

Reply:
The terminology itself is already broken, talking about url/path instances while in fact they are just functions returning an instance of another class.

My feeling: if you're new to Django, you should (probably) be using `path`, and when you are using `re_path` (the new `url`), or even some other magic, you are probably already advanced enough that you can deal with the error message being a bit simplified.